eth/tracers/internal/tracetest/calltrace_test.go

```diff
@@ -176,6 +175,7 @@ func testCallTracer(tracerName string, dirPath string, t *testing.T) {
 	}
 }
 
+// TODO (@sina) this benchmark has been broken for a while, please fix it.
```

Comment:
@s1na  could you please check out this failing test?

Reply:
I have fixed the benchmark